src/app/components/client/settings/ButtonAddEmail.tsx

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ReactNode, useEffect, useState } from "react";
+import { getL10n } from "../../../functions/server/l10n";
+
+export type Props = {
+  children: ReactNode;
+};
+
+const onClick = () => {
+  // FOO
```

Comment:
suggestion/nit: remove `onClick()` functionality?

Reply:
Oh heh, this entire component is unused :) Removed in 41c0eb8c3